lib/targetHandler.js

```diff
@@ -41,7 +41,7 @@ const getCriTargets = async function (targetUrl) {
         response = { matching: [], others: [] };
         for (const target of targets) {
             target.url = handleUrlRedirection(target.url);
-            if (target.title === targetUrl || target.url.split('://')[1] === (targetUrl.split('://')[1] || targetUrl)) {
+            if (target.title === targetUrl || target.url.split('://')[1] === (targetUrl.split('://')[1] || targetUrl) || target.url.split('://')[1] === (targetUrl.split('://www.')[1] || targetUrl)) {
```

Comment:
There should be a better way to do this. We can compare against the domain, path of the url to achieve this.